public/pages/Channels/Channels.tsx

```diff
@@ -215,74 +220,121 @@ export class Channels extends MDSEnabledComponent<ChannelsProps, ChannelsState>
+      items={this.state.items}
+      setItems={(items: ChannelItemType[]) => this.setState({ items })}
+      refresh={this.refresh} />;
+
+    const channelControlsComponent = <ChannelControls
+      onSearchChange={this.onSearchChange}
+      filters={this.state.filters}
+      onFiltersChange={this.onFiltersChange} />;
+
+    const basicTableComponent = <EuiBasicTable
+      columns={this.columns}
+      items={this.state.items}
+      itemId="config_id"
+      isSelectable={true}
+      selection={selection}
+      noItemsMessage={<EuiEmptyPrompt
+        title={<h2>No channels to display</h2>}
+        body="To send or receive notifications, you will need to create a notification channel."
+        actions={<EuiSmallButton href={`#${ROUTES.CREATE_CHANNEL}`}>
+          Create channel
+        </EuiSmallButton>} />}
+      onChange={this.onTableChange}
+      pagination={pagination}
+      sorting={sorting}
+      tableLayout="auto"
+      loading={this.state.loading} />;
+
     return (
       <>
-        <ContentPanel
-          actions={
-            <ContentPanelActions
-              actions={[
-                {
-                  component: (
-                    <ChannelActions
-                      selected={this.state.selectedItems}
-                      setSelected={(selectedItems: ChannelItemType[]) =>
-                        this.setState({ selectedItems })
-                      }
-                      items={this.state.items}
-                      setItems={(items: ChannelItemType[]) =>
-                        this.setState({ items })
-                      }
-                      refresh={this.refresh}
-                    />
-                  ),
-                },
-                {
-                  component: (
-                    <EuiSmallButton fill href={`#${ROUTES.CREATE_CHANNEL}`}>
-                      Create channel
-                    </EuiSmallButton>
-                  ),
-                },
-              ]}
-            />
-          }
-          bodyStyles={{ padding: 'initial' }}
-          title="Channels"
-          titleSize="m"
-          total={this.state.total}
-        >
-          <ChannelControls
-            onSearchChange={this.onSearchChange}
-            filters={this.state.filters}
-            onFiltersChange={this.onFiltersChange}
-          />
-          <EuiHorizontalRule margin="s" />
+        {getUseUpdatedUx() ? (
+          <ContentPanel
+            actions={
+              <ContentPanelActions
+                actions={[
+                  {
+                    component: (
+                      <PageHeader
+                        appRightControls={headerControls}
+                        appLeftControls={[
+                          { renderComponent: totalChannels },
+                        ]}
+                      />
+                    ),
+                  }
+                ]}
+              />
```

Comment:
This might be redundant since, in this case we are not rendering anything inside the actions. We should put PageHeader outside the actions

```
<>
  { getUseUpdatedUx() ? (
    <>
        <PageHeader>...
        <ContentPanel>
...
```